srcpkgs/runit-void/template

```diff
@@ -27,10 +27,39 @@ make_dirs="
  /etc/zzz.d/resume 0755 root root"
 
 post_install() {
+	local arch="x86_64" libc="glibc" float
+
+	case "$XBPS_TARGET_MACHINE" in
+		*-musl) libc="musl";;
+		*) libc="glibc";;
+	esac
+
+	case "$XBPS_TARGET_MACHINE" in
+		i686*) arch="x86_32";;
```

Comment:
Why is this different here than everywhere else in void?

Reply:
well, we can go with i686 of course, but kernel slang is i386 which we don't support. x86 or x86_32 are synonymous for i?86 32 bit AFAIK.

Reply:
@Vaelatern using i686 now